src/main/java/dp/healthcheck/Health.java

```diff
@@ -0,0 +1,16 @@
+package dp.healthcheck;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class Health {
+
+    @RequestMapping("/health")
+    HttpStatus healthCheck() {
+        return HttpStatus.OK;
+    }
+
+}
```

Comment:
Are we going to migrate to use this endpoint as part of this task?

Reply:
This wasn't part of the task, I can remove it if required. But I did it for consistency with other services. Or shall I create a separate PR?